desktop/apps/auction2/works.js

```diff
@@ -2,11 +2,14 @@ import React from 'react'
 import ArtworksContainer from './components/auction_artworks'
 import HeaderContainer from './components/header'
 
-const Works = () => (
+export const Works = () => (
   <div>
-    <HeaderContainer />
     <ArtworksContainer />
   </div>
 )
 
-export default Works
+export const Header = () => (
+  <div>
+    <HeaderContainer />
+  </div>
+)
```

Comment:
This is seemingly super minor, but is an important habit when working with React (or UI in general) since how your filesystem is structured is almost equally as important as how your code is structured -- one component per file. Files are cheap, and being able to see whats going on just by seeing whats going on in your file tree is nice, and helps as things scale : ) It's totally ok to have tiny components.